torch/csrc/jit/tensorexpr/codegen.cpp

```diff
@@ -48,11 +48,19 @@ std::unique_ptr<CodeGen> CreateCodeGen(
   return method(stmt, params, device, kernel_func_name);
 }
 
+ExprHandle expr_to_vec(ExprHandle v, int lanes) {
```

Comment:
This is almost copied from https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/tensorexpr/llvm_codegen.cpp#L477. What's the best place to put a utility function like this, if two similar callsites warrants refactoring?

Reply:
I could see this going in any of codegen, ir, or expr.  Maybe codegen since that's where it's getting used?